HelpSource/Classes/HenonC.schelp

```diff
@@ -43,17 +43,13 @@ Second value of x
 examples::
 code::
 // default initial params
-~defaultParms={ HenonC.ar(MouseX.kr(20, SampleRate.ir)) * 0.2 !2};
+{ HenonC.ar(MouseX.kr(20, SampleRate.ir)) * 0.2 !2}.scope(s);
```

Comment:
better formatting: `0.2 ! 2 }.scope(s);` (spaces between operands and operators)